bin/tmt

```diff
@@ -1,34 +1,28 @@
 #!/usr/bin/python
 
+import functools
+import sys
+
 import click
 
 import tmt.cli
 import tmt.utils
 
-
-def show_exception(exception: BaseException) -> None:
-    """ Show exception in red, handle exception chain """
-    click.echo(click.style(str(exception), fg='red'), err=True)
-
-    # Mention also the original exception if provided
-    if exception.__cause__:
-        click.echo("The exception was caused by the previous exception:", err=True)
-        show_exception(exception.__cause__)
+# A small helper for "print to sys.stderr".
+OUT = functools.partial(print, file=sys.stderr)
```

Comment:
suggestion: if we are here, could we rename the helper function to `STDERR`? makes it more clear to me what it does as `OUT`

Reply:
My original name was `print_error`, but then I saw `print_error()` calls (no arguments) which add some empty lines as a formatting help, and I was thiking, it's called "print_error" but it's called without any error, that's weird... So I went for `OUT` which is more related to `print`, and seeing it without any arguments wasn't that puzzling. So I'd be worried that `STDERR()` could be also suspicious, although not as much as the original "print error".

Reply:
yeah, naming :) it was just a suggestion, I leave it to other reviewers to share the opinion

Reply:
Right, a missing final exception in the log. Good point, but would you mind if I add it in a follow-up patch?

Reply:
Sure. This PR keeps status quo.

Reply:
Hm, naming is hard ;) I must admit that `OUT` is super confusing to me: At the first sight I think to myself what a weird constant name? And it's called? It's a function? Why function in capital letters? And what is it outing? ...

@happz, your first thought seems much better to me. Or, perhaps, what about `print_stderr()`?

Reply:
Luckily for us, while naming is hard, renaming is pretty cheap. Changed back to `print_error()`, waiting eagerly for reviews.